src/spatialdata_plot/pl/utils.py

```diff
@@ -475,12 +625,19 @@ def _map_color_seg(
 
     if seg_erosionpx is not None:
         val_im[val_im == erosion(val_im, square(seg_erosionpx))] = 0
+
     # check if no color is assigned, compute random colors
     unique_cols = np.unique(cols)
     if len(unique_cols) == 1 and unique_cols == 0:
         RNG = default_rng(42)
         cols = RNG.random((len(cols), 3))
 
+    # add alpha value to all colors ([0.1, 0.2, 0.3] -> [0.1, 0.2, 0.3, alpha])
+    target_alpha = 0.5
+    arr = np.array(cols)
+    new_col = np.full((arr.shape[0], 1), target_alpha)
+    cols = np.concatenate((arr, new_col), axis=1)
```

Comment:
why is this needed?

Reply:
Hoped at that point that it could convince the label2rgb function to produce transparent labels, turns out it doesn't. Removed it now 👍